lib/conjur/api/groups.rb

```diff
@@ -33,5 +32,9 @@ def create_group(id, options = {})
     def group id
       standard_show Conjur::Core::API.host, :group, id
     end
+
+    def find_groups options
```

Comment:
A doc string about supported options would be super helpful. Generally it sounds like we want to be better about docing the APIs